spec/integration/_dev/deploy/agent/custom-agent.spec.yml

```diff
@@ -13,23 +13,58 @@ spec:
         anyOf:
           - required:
             - hostname
+    volume:
+      type:
+        - object
+        - "null"
+      properties:
+        name:
+          type: string
+        driver:
+          type: string
+        driver_opts:
+          type: object
+          patternProperties:
+            "^.+$": 
+              type:
+                - string
+                - number
+        external:
+          type:
+            - boolean
+            - object
+          properties:
+            name:
+              deprecated: true
+              type: string
+          additionalProperties: false
+          patternProperties: { "^x-": {} }
+      additionalProperties: false
+      patternProperties: { "^x-": {} }
   properties:
     version:
       description: Docker Compose version
       type: string
       pattern: '^2.3$'
-    services:
-      description: Service list
+    volumes:
```

Comment:
Could you add an example of a volume in a test package? For example in `./test/packages/deploy_custom_agent/_dev/deploy/agent/custom-agent.yml`.